source/versionInfo.py

```diff
@@ -6,6 +6,11 @@
 
 import os
 
+version_year=2016
+version_major=3
+version_minor=0
+version_build=0

```

Comment:
nit: I wonder if these should be moved down to where version is defined, just to keep them all in the same place.